### DISCUSSION TITLE
Dumb new user installation question

### DISCUSSION THREAD

**Discussion by doobes** [music-assistant/support#1771] [Q&A] [answered]
Greetings,

Fairly new to HA and definitely new to MA.

I've gone through the process of installing MA, but it doesn't appear in the list in the right hand pane.

MA is shown in the integrations page with 11 devices, but that's all I've got.

I've rebooted HA a couple of times as suggested and cleared my cache to no avail.

Any thoughts?

chris

**Comment by doobes** [comment]
Sorry, prolly need further info.

I'm running HA on a Home Assistant yellow device. The Rpi CM has 32gb of onboard storage with 4 gb for memory.

Thanks

**Comment by OzGav** [comment, marked as answer]
Second last FAQ https://github.com/orgs/music-assistant/discussions/710

**Reply by doobes** [reply]
Thank you!
